scenarios/traces/es-jaeger/helm/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v2
+name: tracing-es-jaeger
+description: A Helm chart for tracing-es-jaeger, benchmarking scenario
+home: https://github.com/timescale/helm-charts
+sources:
+  - https://github.com/timescale/helm-charts
+maintainers:
+  - name: timescale
+    url: https://www.timescale.com/
+keywords:
+  - observability
+  - monitoring
+  - tracing
+  - opentelemetry
+  - benchmarking
```

Comment:
nit: those are optional fields and not needed if we just use `Chart.yaml` for depenedency management.

Reply:
👍 I'll clean it up.